README.md

```diff
@@ -29,6 +29,7 @@ Need help? Try #logstash on freenode IRC or the https://discuss.elastic.co/c/log
 - Install dependencies
 ```sh
 bundle install
+rake install_jars
```

Comment:
You should run `bundle exec rake install_jars` if you want the rake command to run in the context of the current bundle.

Reply:
Perfect! that's probably why my dependency conflicts weren't caught before Travis tried running